src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -391,12 +391,14 @@ def setupProcessingTask(self, procTask, taskType, inputDataset=None, inputStep=N
         procTaskCmsswHelper.setOverrideCatalog(self.overrideCatalog)
         procTaskCmsswHelper.setErrorDestinationStep(stepName=procTaskLogArch.name())
 
-        if forceMerged:
+        if forceMerged and isinstance(forceMerged, bool):
```

Comment:
That check is actually not doing anything for now. I just put it in since the corresponding check for the forceUnmerged does something and distinguishes between bool and list. Figured I might as well check if forceMerged is a bool and not some random type.

Effectively this specifically checks for and ignores any type but bool and list for forceUnmerged. Figured I might as well check for and ignore any type but bool for forceMerged.